nitransforms/nonlinear.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, field, reference=None):
         self._field = field.get_data()
 
         ndim = self._field.ndim - 1
-        if self._field.shape[:-1] != ndim:
+        if len(self._field.shape[:-1]) != ndim:
```

Comment:
@oesteban is this the intended behavior?

Reply:
I'm guessing so.

Reply:
I'm guessing so :)